<launch>
    <arg name="fiducial_len" default="0.093" />
    <arg name="fiducial_id" default="10" />

    <include file="$(find aruco_detect)/launch/aruco_detect.launch" >
        <arg name="camera" value="camera/rgb" />
        <arg name="image" value="image_raw" />
        <arg name="fiducial_len" value="$(arg fiducial_len)" />
    </include>

    <node pkg="lemonbot_calibration" type="fiducial_broadcaster">
        <param name="base_frame_id" value="lemonbot_camera_rgb_optical_frame" />
        <param name="object_frame_id" value="lemonbot_calibration_object" />
        <param name="fiducial_id" value="$(arg fiducial_id)" />
    </node>

    <node pkg="visp_hand2eye_calibration" type="visp_hand2eye_calibration_calibrator" name="calibrator_node" />
</launch>